src/engraving/dom/capo.cpp

```diff
@@ -53,13 +53,15 @@ PropertyValue Capo::getProperty(Pid id) const
         return m_params.fretPosition;
     } else if (id == Pid::CAPO_IGNORED_STRINGS) {
         std::vector<int> ignoredStrings;
+        ignoredStrings.reserve(m_params.ignoredStrings.size());
         for (string_idx_t string : m_params.ignoredStrings) {
             ignoredStrings.push_back(static_cast<int>(string));
         }
-
         return ignoredStrings;
     } else if (id == Pid::CAPO_GENERATE_TEXT) {
         return m_shouldAutomaticallyGenerateText;
+    } else if (id == Pid::CAPO_TRANSPOSE_MODE) {
+        return static_cast<int>(m_params.transposeMode);
```

Comment:
I know this will be a bit boring to do, but we try as much as possible to avoid read/write of enum properties as integers, because it makes it harder (and dangerous) to make any changes to the enum in future. Instead, we create a dedicated property type, implement conversions for read/write in typesconv.cpp, and for PropertyValue in propertyvalue.cpp. For a good example of how we do it see e.g. AutoOnOff. This means you will never need to explicitely convert the enum to integers.